ui/src/SyntaxResult.js

```diff
@@ -41,13 +41,21 @@ export default function SyntaxResult({ content, status }) {
           ".MuiTreeItem-content.Mui-expanded .subcaption" : { visibility: "visible" },
           "table": { borderCollapse: 'collapse', fontSize: '80%' },
           "td, th": { padding: '0.2em 0.5em', verticalAlign: 'top' },
-          ".pre": { whiteSpace: 'pre', display: 'block' },
+          ".pre": {
+            whiteSpace: 'pre-wrap',
+            wordBreak: 'break-word',
+          },
           ".mono": { fontFamily: 'monospace, monospace', marginTop: '0.3em' }
         }}
       >
         <TreeItem nodeId="0" label="Syntax">
         { rows.length
             ? rows.map(item => {
+              const msg_parts = item.msg.split('\n')
+              const whitespaces = msg_parts[msg_parts.length -1].match(/\s*/)[0].length;
+              const error_instance = msg_parts[msg_parts.length - 2]
+              const modifiedStr = `${error_instance.substring(0, whitespaces)}<span style='text-decoration:underline; font-weight:bold; background-color:#ddd;'>${error_instance[whitespaces]}</span>${error_instance.substring(whitespaces + 1)}`;
```

Comment:
The token length is not necessarily 1. Try one with a duplicate numeric instance id. It will highlight the entire definition I think. (Not sure actually abt other cases of actual tokens if length can be more than one. )

```
#1=IFCPERSON(...
#1=IFCWALL(...
```

But don't assume +1, but count the number of `^` characters.

It would also make the regex a little bit more robust (making sure there is actually a `^`  in there) by combining those:

Something like `const [numWhitespace, numHighlight] = .match(/(\s*)(\^+)).map(s => s.length)`

Reply:
Also missing `;`

Reply:
Maybe also try with a `border:solid 1px red` for the span style to make it even clearer.

Reply:
Done https://github.com/Ghesselink/ifc-pipeline-validation/commit/2634fe16d23e3a951f6fc8c8fac0d38559cd3858

This worked 
` const [_, numWhitespace, numHighlight] = msg_parts[msg_parts.length -1].match(/(\s*)(\^+)/).map(s => s.length);`

`border:solid 1px red`
Is also added 

The result. Clearer indeed :)
![image](https://github.com/AECgeeks/ifc-pipeline-validation/assets/54070862/02d97a32-9a6e-43b9-a9c0-a5f1d7ae72f7)